src/replication/adapter/quay/adapter.go

```diff
@@ -223,3 +257,35 @@ func (a *adapter) getNamespace(namespace string) (*model.Namespace, error) {
 		Name: namespace,
 	}, nil
 }
+
+// PushManifest ...
+func (a *adapter) PushManifest(repository, reference, mediaType string, payload []byte) error {
+	err := a.Adapter.PushManifest(repository, reference, mediaType, payload)
+	if err != nil {
+		if comErr, ok := err.(*common_http.Error); ok {
+			if comErr.Code == http.StatusAccepted {
+				return nil
+			}
+		}
+	}
+	return err
+}
+
+// PullBlob ...
+func (a *adapter) PullBlob(repository, digest string) (size int64, blob io.ReadCloser, err error) {
+	size, blob, err = a.Adapter.PullBlob(repository, digest)
+	if err != nil && blob != nil {
+		if size == 0 {
```

Comment:
Why ignore error?

Reply:
In native PullBlob, it will read 'Content-Lenght' in the response header, but it has no header of 'Content-Lenght' return when we use enterprise quay, so it will return an error due to converting the empty string to an integer type, here ignore the error when the blob is not nil and ensure it's can be working.

Reply:
I think quota compute will also have problem if no size. 
have a look @heww @wy65701436